domain/consensus/processes/ghostdagmanager/ghostdagmanager.go

```diff
@@ -29,9 +28,3 @@ func New(
 		k:                  k,
 	}
 }
-
-// BlockData returns previously calculated GHOSTDAG data for
-// the given blockHash
-func (gm *ghostdagManager) BlockData(blockHash *externalapi.DomainHash) (*model.BlockGHOSTDAGData, error) {
-	return gm.ghostdagDataStore.Get(gm.databaseContext, blockHash)
```

Comment:
Does it mean every caller will have to have its own databaseContext?

Reply:
Yeah.